test/functional/wallet_bumpfee.py

```diff
@@ -105,18 +105,18 @@ def run_test(self):
 
     def test_invalid_parameters(self, rbf_node, peer_node, dest_address):
         self.log.info('Test invalid parameters')
-        rbfid = spend_one_input(rbf_node, dest_address)
+        rbfid = spend_one_input(rbf_node, dest_address, data='deadbeef')
```

Comment:
In 5068940043a4c44bb9e7d3a2d70a12c753ff82e4 "test: add tests for `outputs` argument to `bumpfee`/`psbtbumpfee`"

It's not obvious to me why this test changed? I don't see how adding a data carrier output to this test is related to the rest of the changes in this PR.